style.css

```diff
@@ -0,0 +1,481 @@
+/* stylelint-disable */
+
+
+/* STYLE SHEET */
+
+
+/* GLOBALS */
+
+body {
+    max-width: 144rem;
+    margin: 0 auto;
+}
+
+html {
+    box-sizing: border-box;
+    font-size: 62.5%;
+    font-family: 'Roboto', sans-serif;
+    /* 1rem = 10px */
+}
+
+h1 {
+    font-size: 4rem;
+}
+
+h2 {
+    font-size: 4.5rem;
+    margin: 0;
+}
+
+*,
+*::before,
+*::after {
+    box-sizing: inherit;
+}
+
+img {
+    max-width: 100%;
+    height: auto;
+}
+
+ul {
+    margin: 0;
+}
+
+.d-flex {
+    display: flex;
+}
+
+.d-grid {
+    display: grid;
+}
+
+.flex-space {
+    justify-content: space-between;
+}
+
+.main-container {
+    position: relative;
+}
+
+
+/*  HEADER */
+
+.navbar {
+    background-color: #333;
+    color: #fff;
+    padding: 0 0.8rem;
+}
+
+ul.d-flex {
+    list-style: none;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.d-flex li {
+    padding: 2rem;
+    font-size: 2rem;
+    font-weight: bold;
+    background-color: #8b8b8b;
+}
+
+.searh-top {
+    justify-content: space-around;
+}
+
+.logo-nav {
+    width: 4rem;
+}
+
+.bg-white {
+    background-color: #fff;
+}
+
+.bg-black {
+    background-color: #000;
+}
+
+
+/* --------------------- */
+
+
+/* MAIN CONTENT */
+
+.page-title {
+    position: absolute;
+    left: 25%;
+    top: 3rem;
+    font-family: sans-serif;
+    font-weight: bolder;
+    width: 50%;
+    text-align: center;
+}
+
+.page-title span {
+    padding: 0.8rem 1rem;
+}
+
+.grid-container {
+    grid-template: repeat(2, 1fr) / repeat(2, 1fr);
+    align-self: center;
+}
+
+.grid-child {
+    height: 70rem;
+    justify-content: center;
+    align-items: center;
+}
+
+.grid-content {
+    color: #fff;
+    width: 75%;
+    height: 35rem;
+    margin-top: 3rem;
```

Comment:
You are using this property 9 times throughout your CSS, maybe it would be better to make an `mt-3` class? :wink:

Reply:
FIxed